src/Toolkit/File.php

```diff
@@ -72,11 +72,16 @@ public function copy(string $target, bool $force = false)
     /**
      * Returns the file as data uri
      *
+     * @param bool $base64 Whether the data should be base64 encoded or not
      * @return string
      */
-    public function dataUri(): string
+    public function dataUri(bool $base64 = true): string
     {
-        return 'data:' . $this->mime() . ';base64,' . $this->base64();
+        if ($base64 === true) {
+            return 'data:' . $this->mime() . ';base64,' . $this->base64();
+        }
+
+        return 'data:' . $this->mime() . ',' . $this->read();
```

Comment:
Wouldn't this need a run through `rawurlencode()` to produce valid output?

Reply:
I'm afraid you have more knowledge here than me... so not sure.